packages/fxa-payments-server/src/components/FetchErrorDialogMessage.tsx

```diff
@@ -12,14 +12,18 @@ const FetchErrorDialogMessage = ({
   title: string;
   testid?: string;
   fetchState: FetchState<any, APIError>;
-  onDismiss?: Function;
+  onDismiss?: () => void;
 }) => (
-  <DialogMessage className="dialog-error" onDismiss={onDismiss}>
+  <Modal
+    id="dialogs"
+    className="dialog-error"
+    onDismiss={onDismiss}
```

Comment:
Sorry, I think I missed seeing this bit in Portal:

```
    if (id !== 'alert-bar-portal') {
      el.setAttribute('role', 'dialog');
    }

    if (id === 'modal') {
      el.setAttribute('aria-labelledby', headerId);
      el.setAttribute('aria-describedby', descId);
      setA11yOnAdjacentElementsAndBody(
        document.querySelectorAll(TOP_LEVEL_NONMODAL_DIVS_SELECTOR)
      );
    }
```

Maybe there is a distinction between what we are calling a "modal" and what we are calling a "dialog message" then since we don't apply these aria labels if `id = "dialogs"`. I'll have to check back into your branch later and spin up payments to poke around.

Reply:
I'm aligned with keeping `modal`/`modal-content-container`, etc, and dropping `id = dialogs` as well to clean everything up. I wasn't sure if there was a reason `DialogMessage` didn't have `headerId` and `descId`, and why `Portal` specified to only kept info if it had `id === modal` (see below).

```
if (el && el.children.length === 1) {
    // Reset any non-portal properties here
    if (id === 'modal') {
      // When unloaded, we do not remove the portal element in order to allow
      // a series of portal dependent components to be rendered.
      resetA11yOnAdjacentElementsAndBody(
        document.querySelectorAll(TOP_LEVEL_NONMODAL_DIVS_SELECTOR)
      );
    } else {
      el.remove();
    }
  }
```

If `Portal` is used properly, we would be able to unload everything and keep the a11y wins since `<div id="portal"></div>` would be permanent. I feel like there is a way to include `aria-labelledby`/`aria-describedby` without DOM manipulation, I'll mull it over.

Reply:
If I update `FetchErrorDialogMessage`, should I update the following as well?:
- SuccessDialog
- ConfirmationDialog
- PlanErrorDialog